src/main/java/mate/academy/spring/controller/UserController.java

```diff
@@ -0,0 +1,65 @@
+package mate.academy.spring.controller;
+
+import java.util.List;
+import java.util.NoSuchElementException;
+import java.util.stream.Collectors;
+import mate.academy.spring.model.User;
+import mate.academy.spring.model.dto.UserResponseDto;
+import mate.academy.spring.service.UserService;
+import mate.academy.spring.service.mapper.UserDtoMapper;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/users", method = RequestMethod.GET)
```

Comment:
It seems to me that we do not need to specify the method here 
```suggestion
@RequestMapping("/users")
```